docs/05-infrastructure-nodes.md

```diff
@@ -54,12 +54,15 @@ metadata:
     sigs.k8s.io/cluster-api-machine-role: worker
     sigs.k8s.io/cluster-api-machine-type: worker
   name: 190125-3-worker-us-west-1b
-  namespace: openshift-cluster-api
+  namespace: openshift-machine-api
   resourceVersion: "9027"
-  selfLink: /apis/cluster.k8s.io/v1alpha1/namespaces/openshift-cluster-api/machinesets/190125-3-worker-us-west-1b
+  selfLink: /apis/cluster.k8s.io/v1alpha1/namespaces/openshift-machine-api/machinesets/190125-3-worker-us-west-1b
```

Comment:
should be "/apis/machine.openshift.io/v1beta1/namespaces/openshift-machine-api/machinesets/190125-3-worker-us-west-1b"

Reply:
fixed at some point